akka-remote/src/main/resources/reference.conf

```diff
@@ -268,465 +268,467 @@ akka {
 
 akka {
 
-  remote.classic {
-    #//#classic
+  remote {
+  #//#classic
+    classic {
```

Comment:
Good to clarify this. We could even move the entire section to the end of the file.
```
#//#classic
akka.remote.classic {
  ...
}
#//#classic

Reply:
I think I was wrong about this